dependency-lint.yml

```diff
@@ -0,0 +1,46 @@
+# array of strings or regular expressions to match against a module name
+# to determine if it is allowed to be unused
+allowUnused:
+  - cucumber
+  - livescript
+
+# array of path patterns to match againt a filename
+# to determine if it is used only for development
+#   (see https://github.com/isaacs/minimatch)
+devFilePatterns:
+  - '{features,spec,test}/**/*'
+  - '**/*_{spec,test}.js'
+
+# array of strings or regular expressions to match against a script name
+# in your `package.json` to determine if it is used only for development
+devScripts:
+  - lint
+  - publish
+  - test
+  - version
+
+# path pattern to match against a filename to determine if it should be parsed.
+# This is the starting point, devFilePatterns and ignoreFilePatterns should be
+# subsets of this pattern
+#   (see https://github.com/isaacs/minimatch)
+filePattern: '**/*.ls'
+
+# array of path patterns to match against a filename
+# to determine if it should be ignored
+#   (see https://github.com/isaacs/minimatch)
+ignoreFilePatterns:
+  - 'node_modules/**/*'
```

Comment:
This line is generated by dependency-lint, so it is smart enough to know, but I can't think of any reason why someone would want to remove this line, so maybe it should just automatically ignore it.

@charlierudolph what do you think?